docs/src/lib/flattenTokenTree.js

```diff
@@ -16,7 +16,7 @@ export function flattenTokenTree( nestedTokens ) {
 
 		return tokens.concat( {
 			name: node.name,
-			referencedTokens: node.original.value.replace( /{|}|.value/gi, '' ),
+			referencedTokens: node.attributes.tokens.join( ', ' ),
```

Comment:
I'm a bit unsure if gluing this by comma here is going too far into the presentation territory already (could be done just as well in the respective story) OTOH it deduplicates which is good. And it is currently a no-op anyway (no situations of two tokens informing another AFAICT).